### PR TITLE
Reduce package manager scaffolding.

### PR DESCRIPTION
**Pull request by jdalton**
This PR tackles #1175.

### CODE REVIEW

File: support/xyz.sh [hunk 1]
```diff
@@ -160,9 +158,6 @@ inc() {
 
 inc package.json
 
-run "$dir/sync-package-managers.js"
-run "git add --force *.json"
```
Comment (aearly) [inline support/xyz.sh]: Would we still need this line to stage changes to the `package.json`?

**Comment by aearly** [comment]
I love PRs that remove 150 lines :smile:

File: support/xyz.sh (hunk 1 above)
Comment (jdalton) [inline support/xyz.sh]: Fixed.

**Comment by megawac** [comment]
+++

**Pull request merged**